<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>721</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="tab1_content_layout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Добавить данные</string>
         </property>
         <widget class="QLabel" name="dateLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>120</width>
            <height>13</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Дата</string>
          </property>
         </widget>
         <widget class="QDateEdit" name="addDateEditField">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>120</width>
            <height>20</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="typeLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>18</width>
            <height>13</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Тип</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="incomesRadioButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>63</width>
            <height>17</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>доходы</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="expensesRadioButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>расходы</string>
          </property>
         </widget>
         <widget class="QLabel" name="sumLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>210</y>
            <width>31</width>
            <height>13</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Сумма</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="sumEditField">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>230</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QWidget" name="">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>140</y>
            <width>121</width>
            <height>49</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="categoryLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Категория</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="categoryToolButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QComboBox" name="categoryComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="rowCount">
          <number>2</number>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>100</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>25</number>
         </attribute>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="rightVerticalLayout">
         <item>
          <widget class="QGroupBox" name="showCheckBoxGroup">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>80</height>
            </size>
           </property>
           <property name="title">
            <string>Показывать</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <widget class="QCheckBox" name="incomesCheckBox">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>20</y>
              <width>70</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>доходы</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="expensesCheckBox">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>50</y>
              <width>70</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>расходы</string>
            </property>
           </widget>
           <zorder>incomesCheckBox</zorder>
           <zorder>expensesCheckBox</zorder>
           <zorder>chooseDateGroup</zorder>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="intervalRadioButtonGroup">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>80</height>
            </size>
           </property>
           <property name="title">
            <string>Период</string>
           </property>
           <widget class="QRadioButton" name="dayRadioButton">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>20</y>
              <width>82</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>День</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="monthRadioButton">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>40</y>
              <width>82</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Месяц</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="yearRadioButton">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>60</y>
              <width>82</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Год</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="chooseDateGroup">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>80</height>
            </size>
           </property>
           <property name="title">
            <string>Дата</string>
           </property>
           <widget class="QDateEdit" name="showDateEditField">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>131</width>
              <height>22</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentSection">
             <enum>QDateTimeEdit::DaySection</enum>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="previousDateButton">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>41</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
           <widget class="QPushButton" name="nextDateButton">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>50</y>
              <width>41</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <zorder>tableWidget</zorder>
     <zorder>chooseDateGroup</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>showCheckBoxGroup</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>showCheckBoxGroup</zorder>
     <zorder>tableWidget</zorder>
     <zorder>chooseDateGroup</zorder>
     <zorder>intervalRadioButtonGroup</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
